CMakeLists.txt

```diff
@@ -844,6 +872,8 @@ if( PETSC_FOUND )
   
   
   ### add target for flow123d binary
+  #
+  # set(CMAKE_EXECUTABLE_FORMAT "ELF")
```

Comment:
There is/was [bug](https://cmake.org/Bug/view.php?id=13934) in cmake and I was not able to **cpack** any tarball. I found workaround but I don't know nor I'm able to find what does command do:
```cmake
set(CMAKE_EXECUTABLE_FORMAT "ELF")
```
But it helped in my case. Should I uncomment or delete it @jbrezmorf ?